app/styles/partials/nav-bar.scss

```diff
@@ -26,8 +26,13 @@
   .signout {
     padding-left: 70%;
   }
+}
 
-  a.item {
```

Comment:
Don't remove this styling as it may have an impact on other areas of the navbar, instead overwrite any effects it has on the drop down menu in `.menu.profile-dropdown-menu`